recipes/erkir/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ErkirTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#cmake-targets